src/sourmash/cli/utils.py

```diff
@@ -93,3 +94,26 @@ def command_list(dirpath):
     basenames = [os.path.splitext(path)[0] for path in filenames if not path.startswith('__')]
     basenames = filter(opfilter, basenames)
     return sorted(basenames)
+
+
+def check_scaled_bounds(arg):
+    actual_min_val = 0
+    min_val = 100
+    max_val = 1e6
+
+    f = float(arg)
+
+    if f < actual_min_val:
+        raise argparse.ArgumentTypeError(f"ERROR: --scaled value must be positive")
+    if f < min_val:
+        notify('WARNING: --scaled value should be >= 100. Continuing anyway.')
+    if f > max_val:
+        notify('WARNING: --scaled value should be <= 1e6. Continuing anyway.')
+    return f
+
+
+def add_scaled_arg(parser, default=None):
+    parser.add_argument(
+        '--scaled', metavar='FLOAT', type=check_scaled_bounds, default=0,
+        help='scaled value should be between 100 and 1e6; default={d}'.format(d=default)
```

Comment:
two suggestions - 

first, I think printing out the default is not useful, because it contradicts the help message (0 is not between 100 and 1e6!)

second, even if you decide to leave the default in, you can use an f-string, so `help=f'scaled value should be ... {default}`, without using .format.

Reply:
So then, would the line be something like this: `'--scaled', metavar='FLOAT', type=check_scaled_bounds,
        help='scaled value should be between 100 and 1e6; default={d}'`?

Reply:
note the 'f' prefix before the string to get the substitution going.